website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -107,9 +81,7 @@ For example:
 - for actions on a page that require even less visual weight or priority than a secondary button (our default for less important actions)
 - for controlling form fields (such as adding an additional form field)
 
----
-
-## State
+### State
```

Comment:
The images in the State section are a bit pixelated. I recommend working with @cveigt to get higher quality assets exported.

Reply:
Great idea! I created a jira issue to track. https://hashicorp.atlassian.net/browse/HDS-1316